src/lib/shadcnComponents/ui/accordion/accordion-item.svelte

```diff
@@ -12,6 +12,6 @@
   let { value, children, ...rest }: Props = $props();
 </script>
 
-<AccordionPrimitive.Item {...rest} {value} class={cn('border-b', rest.class)}>
+<AccordionPrimitive.Item {...rest} {value} class={cn('border-gray3 border-b', rest.class)}>
```

Comment:
Should this bottom border go to black when in dark intent?

Reply:
I guess it does show up as that gray, so it is ok.

Reply:
in the designs it was the same gray3 in both